### ISSUE TITLE
Closing issues guidelines

### ISSUE THREAD

**Issue by sndsgn** [visceral-tambourine/Ambiguous-Cicada#2]
Closing Issues via Commits
Including a keyword and the issue # in your commit message will close that issue once it is merged into your default branch. This will automatically move the issue to the Done column on your Waffle board.

keywords:
close, closes, closed
fix, fixes, fixed
resolve, resolves, resolved
examples:
same repo:
git commit -m "closes #104"
other repo:
git commit -m "closes username/repo#104"
Pull Requests are issues too, merging a Pull Request will close it.
